SJH/Day7/체육복.js

```diff
@@ -0,0 +1,24 @@
+function solution(n, lost, reserve) {
+    lost.sort((a, b) => a - b);
+    reserve.sort((a, b) => a - b);
+    let participation = n - lost.length;
+    const LostSet = new Set(lost);
```

Comment:
```suggestion
    const lostSet = new Set(lost);
```

카멜케이스로 작성해주세요! (파스칼 케이스로 작성하는 경우는 나중에 클래스를 작성하거나 타입을 정의할 때 정도입니당)